.github/workflows/.test.yml

```diff
@@ -140,6 +143,14 @@ jobs:
           version: ${{ env.SETUP_BUILDX_VERSION }}
           driver-opts: image=${{ env.SETUP_BUILDKIT_IMAGE }}
           buildkitd-flags: --debug
+      -
+        name: Build test image
+        uses: docker/bake-action@v3
```

Comment:
v4 is now available

Reply:
Looks like https://github.com/moby/buildkit/blob/master/.github/dependabot.yml didn't trigger :disappointed_relieved: